sap/5.5/modules/ROOT/pages/sap-connector-examples.adoc

```diff
@@ -690,6 +698,499 @@ output application/java
                                       </mule>
 ----
 
+
+[[manage-transactions]]
+== Manage Transactions
+
+This example performs two calls on standard BAPIs that handle bank records and manages those transactions.
+
+The following screenshots show the Studio flows for this example:
+
+image::managetransactions-flow1.png[Studio flow for the first flow of Managing Transactions]
+
+image::managetransactions-flow2.png[Studio flow for the second flow of Managing Transactions]
+
+image::managetransactions-flow3.png[Studio flow for the third flow of Managing Transactions]
+
+
+=== Configure the First Flow
+
+The first flow retrieves a list of available banks in the United States, filters the first result, and fetches information about the bank. To configure this flow:

```

Comment:
In this scenario, we are not working with transactions as it's not necessary for queries to SAP.